controllers/cartController.js

```diff
@@ -0,0 +1,36 @@
+const cartService = require("../services/cartService");
+const { catchAsync } = require("../utils/error");
+
+const getCartList = catchAsync(async (req, res) => {
+  const result = await cartService.getCartList(req.user.id);
+  return res.status(200).json(result);
+});
+
+const addCartItems = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const productId = req.body.productId;
+  const quantity = req.body.quantity;
+  await cartService.addCartItems(userId, productId, quantity);
+  return res.status(201).json({ message: "ADD_ITEMS_SUCCESS" });
+});
+
+const updateItemQuantity = catchAsync(async (req, res) => {
```

Comment:
```suggestion
const updateCart = catchAsync(async (req, res) => {
```
결국엔 cart를 수정하는 것이므로 적절하게 수정해주세요.

Reply:
네 변경했습니다 !